Gemfile

```diff
@@ -24,9 +19,7 @@ end
 # Tools to run during development
 group :development do
   # Needed for lint:yard:junk task
-  if RUBY_VERSION >= '2.3.0'
-    gem 'yard-junk', '~> 0'
-  end
+  gem 'yard-junk', '~> 0'
```

Comment:
can we look into this, this puts no restrictions on it. (A bug with the way gemfiles work)